### ISSUE TITLE
Track I Build tests

### ISSUE THREAD

**Issue by mpalat** [eclipse-jdt/eclipse.jdt.core#3076]
Check the I build failures - https://download.eclipse.org/eclipse/downloads/index.html

The following components will be tracked:

jdt.apt.pluggable.tests
jdt.apt.tests
jdt.compiler.apt.tests
jdt.compiler.tool.tests
jdt.core.tests.builder
jdt.core.tests.compiler
jdt.core.tests.model
jdt.core.tests.performance

**Comment by mpalat** [comment]
@subyssurendran666 Could you please also sign up for this?

**Comment by subyssurendran666** [comment]
Yes @mpalat

**Comment by mpalat** [comment]
All the assignees, please feel free to opt out in case you are not able to. Took the liberty of assigning people. Thanks!

**Comment by srikanth-sankaran** [comment]
This just appears to put together a mechanism - more important I feel is the process. So with seven assignees, what is the recommended process ? Would all seven be looking at I builds every day and triage any issues ? That sounds like duplication of effort on the one hand and an unclear work assignment on the other.

How about we come up with a weekly primary owner for tracking build issues and incoming defect reports ? That person can involve others selectively as needed ?

**Comment by mpalat** [comment]
> How about we come up with a weekly primary owner for tracking build issues and incoming defect reports ? That person can involve others selectively as needed ?

@srikanth-sankaran  Yes, agree. I can volunteer to be the primary owner for this week [Tue, Mon] - please note the [] brackets - both days inclusive :). Someone else can volunteer for the next week starting next Tuesday. what say?

**Comment by srikanth-sankaran** [comment]
> > How about we come up with a weekly primary owner for tracking build issues and incoming defect reports ? That person can involve others selectively as needed ?
> 
> @srikanth-sankaran Yes, agree. I can volunteer to be the primary owner for this week [Tue, Mon] - please note the [] brackets - both days inclusive :). Someone else can volunteer for the next week starting next Tuesday. what say?

That sounds good. I would not burden volunteers (@stephan-herrmann for example) - except when they are needed to be involved as per initial analysis.

Also would exclude those whose have cross project responsibilities (@iloveeclipse for example).

We should also agree on the responsibilities:

    - Tracking build/test problems and raising follow up issues.
    - Tracking and triaging incoming defect reports
    - what else ?

**Comment by srikanth-sankaran** [comment]
I am happy to participate and take weekly ownership on a rotational basis - Nevertheless, I will remove my name from assignees of this ticket, so this doesn't clutter my queries.

**Comment by mpalat** [comment]
> That sounds good. I would not burden volunteers (@stephan-herrmann for example) - except when they are needed to be involved as per initial analysis.
> 
> Also would exclude those whose have cross project responsibilities (@iloveeclipse for example).

Am fine for the above exclusion. However, would leave it to their discretion. Would say this is voluntary rather than mandatory. 

> We should also agree on the responsibilities:
> 
> ```
> - Tracking build/test problems and raising follow up issues.
> - Tracking and triaging incoming defect reports
> - what else ? 
> ```

I have created a label "[build test failure](https://github.com/eclipse-jdt/eclipse.jdt.core/labels/build%20test%20failure)" which is slightly less "serious" than the label "regression" which says "Something was broken by a previous change". would propose to assign this label to the occasional build test failures. This can be an additional task.

Tracking and triaging incoming defect reports is the inbox tracking category. Would you prefer the same person to do both during the same time.?
See issue #3081 for the first issue under this category.

**Comment by stephan-herrmann** [comment]
> > That sounds good. I would not burden volunteers (@stephan-herrmann for example) - except when they are needed to be involved as per initial analysis.
> > Also would exclude those whose have cross project responsibilities (@iloveeclipse for example).
> 
> Am fine for the above exclusion. However, would leave it to their discretion. Would say this is voluntary rather than mandatory.

I'll gratefully make use of that option :) 

Of course feel you may notify me on individual issues, when you see a potential connection to my contributions.

**Comment by mpalat** [comment]
@jarthana will be the primary owner for this week. The primary owner's name will be at the end [how do we do it: just unset the name and set the name] so that we don't have to repeatedly pollute the messages here. thanks

**Comment by mpalat** [comment]
we have 31 failures listed at https://download.eclipse.org/eclipse/downloads/drops4/I20241128-0520/testresults/html/org.eclipse.jdt.core.tests.model_ep435I-unit-win32-x86_64-java17_win32.win32.x86_64_17.html - investigating

**Comment by mpalat** [comment]
> we have 31 failures listed at https://download.eclipse.org/eclipse/downloads/drops4/I20241128-0520/testresults/html/org.eclipse.jdt.core.tests.model_ep435I-unit-win32-x86_64-java17_win32.win32.x86_64_17.html - investigating

This was transient - ref [I20241201-1800](https://download.eclipse.org/eclipse/downloads/drops4/I20241201-1800)

**Comment by mpalat** [comment]
issue #3499  raised for- regression found in [I20241230-0520](https://download.eclipse.org/eclipse/downloads/drops4/I20241230-0520/)

**Comment by iloveeclipse** [comment]
Removed out-of-date 4.34 milestone. Please set valid one if needed.

**Comment by mpalat** [comment]
I20250504-1820 has regressions - issue #3970 raised
